tests/pytests/test_issues.py

```diff
@@ -236,3 +236,33 @@ def test_MOD1266(env):
   env.expect('FT.SEARCH', 'jsonidx', 'redis').equal([1L, '1', ['$', '{"t":"Redis"}']])
   conn.execute_command('JSON.SET', '1', '$.t', r'{"inner_t":"Redis"}')
   env.expect('FT.SEARCH', 'jsonidx', '*').equal([0L])
+
+def testMemAllocated(env):
+  conn = getConnectionByEnv(env)
```

Comment:
I think it's a good idea to verify that the test passes on the coordinator and if not try to adjust it.

Reply:
Checked and passed